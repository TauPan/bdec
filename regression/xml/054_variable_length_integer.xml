<protocol>
    <sequence name="blah">
        <field name="a" length="8" type="integer" />
        <field name="b" length="(${a} * 8)" type="integer" />
    </sequence>

</protocol>
