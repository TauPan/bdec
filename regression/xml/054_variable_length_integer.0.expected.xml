
           <blah>
              <a>3</a>
              <b>83</b>
           </blah> 